kinto/plugins/history/listener.py

```diff
@@ -0,0 +1,94 @@
+from kinto.core.utils import instance_uri
+import copy
+from datetime import datetime
+
+
+def on_resource_changed(event):
```

Comment:
Well actually, I've look and it makes the configuration more verbose for users (plugin+listener), and does not bring anything concrete... So I might want to leave it like that.